content/en/docs/Guides/backup/_index.md

```diff
@@ -49,14 +49,14 @@ curl -s -XGET localhost:4001/db/backup?vacuum -o bak.sql
 >Be sure to study the SQLite VACUUM documentation before enabling this feature, as it may alter the backup you receive in a way you do not want.
 
 ## Automatic Backups
-rqlite supports automatically, and periodically, backing up its data to Cloud-hosted storage. To save network traffic rqlite uploads a compressed snapshot of its SQLite database, and will not upload a backup if the SQLite database hasn't changed since the last upload took place. Only the cluster Leader performs the upload.
+rqlite supports automatically, and periodically, backing up its data to S3-compatible Cloud-hosted storage. To save network traffic rqlite uploads a compressed snapshot of its SQLite database, and will not upload a backup if the SQLite database hasn't changed since the last upload took place. Only the cluster Leader performs the upload.
```

Comment:
I never write "cluster Leader". I always just write "Leader".

Reply:
"cluster leader" is existing language :)

But I can change it in this PR

Reply:
Ha! I was afraid of that! Ha ha.

Yeah, it reads badly to me now. Not a big deal.

Reply:
Because it feels superfluous. Also, you can have a leader with a single node system.

So I *tend* to write just Leader.